src/foam/dao/EasyDAO.js

```
@@ -194,6 +194,13 @@ foam.CLASS({
             .build();
         }
 
+        if ( getLifecycleAware() ) {
```

Can we have lifecycleAware before deletedAware? and log a deprecated warning in the deletedAware block.

---

Can both be enabled at the same time? If not, we should throw an error if both are set to true ..

---

Done to both.